client/src/components/Ticketing/ticketingmanager/AdminPurchase/AdminCart.tsx

```diff
@@ -2,7 +2,8 @@ import React, {ReactElement, useState} from 'react';
 import {useLocation, useNavigate} from 'react-router';
 import {EventRow} from './utils/adminCommon';
 import {getDiscountCode} from './utils/adminApiRequests';
-import {SearchIcon, XIcon} from '../../Icons';
+import {HelpIcon, SearchIcon, XIcon} from '../../Icons';
+import {Tooltip} from '@mui/material';
```

Comment:
Should the Tooltip/HelpIcon be added to fee total in the admin cart as well?

Reply:
I had done that at first, but then realized that admin don't really need a fee disclaimer. I'll remove those stale imports.